C++/include/data_structures/PriorityQueue.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+	Priority Queue
+	--------------
+	Implementing the priority queue using max-heap data structure
+	Heap has two properties
+	1. Structural property i.e structurally it is similar to complete binary tree.
+	2. Max-heap property i.e Parent's priority is no smaller than children's priority(can be equal).
+
+	PRIORITY IS THE HIGHEST FOR THE LARGEST VALUE (PRIORITY IS ALWAYS GREATER THAN ZER0)
+	-----------------------------------------------------------------------------------
+
+	Time complexity
+    ---------------
+    O(log N) - for insert(), Where N is number of items in priority queue
+    O(log N) - for remove() , Where N is number of items in priority queue
+    O(1) - for max() , constant amount of time
+    
+    Space complexity
+    ----------------
+    O(N) - Where N is number of items in priority queue
+*/
+#include "PriorityQueue.hpp"
+#include <iostream>
+#include <vector>
+#include <stdlib.h>
+#include <climits>
+
+using namespace std;
+
+/*
+    Constructor
+*/
+PriorityQueue::PriorityQueue() {
+	v.push_back(INT_MIN); // indices of vector start at 1 
+}
+
+void PriorityQueue::heapify_up(int position) {
+	/* Function to restore the max-heap property while inserting new priorities in the priority Queue.
+	   Variable "position" represent the position at which the heap property is violated */
+
+	while (position > 1 && v[(position/2)] < v[position]) {
+
+		swap(v[(position/2)], v[position]);
+		position = position/2;
```

Comment:
- The parenthesis inside `[]` can be removed
- Add a space to the left and right of `/`